app/models/user.rb

```diff
@@ -218,6 +224,26 @@ def otp_verified?(otp)
     save!(validate: false)
   end
 
+  def webauthn_verified?(current_challenge, webauthn_credential)
```

Comment:
I am not sure I under why this method is in `User` model. Would it be better if we asked `WebauthnCredential` object if it was valid/verified? Do we want to ensure that webauthn_credential belongs to the user?